integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/setup/standalone/CcWithPrometheusMetricsEnabled.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com)
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.tests.setup.standalone;
+
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.wso2.choreo.connect.tests.context.CCTestException;
+import org.wso2.choreo.connect.tests.context.CcInstance;
+import org.wso2.choreo.connect.tests.util.ApictlUtils;
+
+import java.util.concurrent.TimeUnit;
+
+public class CcWithPrometheusMetricsEnabled {
+    CcInstance ccInstance;
+
+    @BeforeTest(description = "initialise the setup")
+    void start() throws Exception {
+        ccInstance = new CcInstance.Builder().withNewDockerCompose("cc-prometheus-metrics.yaml")
+                .withNewConfig("prometheus-metrics-enabled.toml")
+                .withVolumeMountDir("metricConfigs")
+                .build();
+        ccInstance.start();
+        ApictlUtils.addEnv("test");
+        ApictlUtils.login("test");
+        TimeUnit.SECONDS.sleep(5);
+
+    }
+
```

Comment:
Are we expecting add a test case here ?
Seems this file is not having a test case....

Reply:
This is just the setup class, tests are in PrometheusMetricEndpointsTestCase class